.github/workflows/ci.yaml

```diff
@@ -58,6 +58,28 @@ jobs:
       CF_SPACE: "design-system"
 
     steps:
+      - name: Pre-deploy checks
+        uses: actions/github-script@v5
+        with:
+          script: |
+
+            // Check that the code to be deployed is the latest code on the branch on GitHub
+            // If you want to rollback changes to prod, consider using a revert commit to main
+            //
+            const upToDate = github.rest.repos.getCommit({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/heads/main',
+              mediaType: {
+                format: 'sha'
+              },
+            }) == context.sha
+
+            if (!upToDate) {
+              core.info(`Workflow run GITHUB_SHA ${context.sha} does not match commit sha of main branch`)
```

Comment:
For debugging purposes, would it be useful to include the SHA we get as the result of the API call as well? Would mean splitting the conditional into its own statement so we capture the returned SHA, but that might also help to improve legibility anyway.

Reply:
Done

Reply:
Made this change, but for some reason I'm getting `[object Object]` instead of anything useful in the output 🤔 Any idea why?